python/ray/data/BUILD

```diff
@@ -5,12 +5,7 @@
 # --------------------------------------------------------------------
 load("//bazel:python.bzl", "py_test_module_list")
 
-SRCS = [] + select({
-    "@bazel_tools//src/conditions:windows": glob([
-        "**/conftest.py",
-    ]),
-    "//conditions:default": [],
-})
+SRCS = glob(["**/conftest.py"])
```

Comment:
Globs are local to the corresponding package, so this is intended to match `/python/ray/data/tests/conftest.py`, right?

Reply:
Yes. The old code looks like copy-paste from //python/ray/tests, it could be more explicit.

Reply:
An alternative way that I could see this working is having a single `confest` target in the root directory that takes care of both.

```bazel
py_library(
  name = "conftest",
  srcs = ["//python/ray/tests/conftest.py"] + glob(["**/conftest.py"]),
  visibility = [
    "//python/ray/tests:__subpackages__",
    "//python/ray/dashboard:__pkg__",
    "//python/ray/data:__pkg__",
  ],
)
```

Other libraries can then depend on `//:conftest` can get both the root conftest and their package-specific conftest.

What do you think about that alternative? Too implicit?